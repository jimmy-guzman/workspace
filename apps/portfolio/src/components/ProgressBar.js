import React from 'react'

const ProgressBar = () => (
  <div className='progress-bar'>
    <div className='indeterminate' />
  </div>
)

export default ProgressBar
