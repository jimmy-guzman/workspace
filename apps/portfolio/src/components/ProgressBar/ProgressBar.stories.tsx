import React from 'react'
import { ProgressBar } from './ProgressBar'

export default {
  title: 'portfolio/components/ProgressBar',
  component: ProgressBar
}

export const normal = () => <ProgressBar />
