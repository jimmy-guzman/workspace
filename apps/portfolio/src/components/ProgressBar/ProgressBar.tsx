import React from 'react'
import { Styled } from './ProgressBar.styles'

export const ProgressBar = () => (
  <Styled.Bar>
    <Styled.Progress />
  </Styled.Bar>
)
