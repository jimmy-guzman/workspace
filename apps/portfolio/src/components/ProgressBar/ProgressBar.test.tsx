import React from 'react'
import { render } from '@testing-library/react'

import { ProgressBar } from './ProgressBar'

const setupProgressBar = () => {
  return render(<ProgressBar />)
}

describe('<ProgressBar />', () => {
  it('should render default snapshot', () => {
    const { container } = setupProgressBar()

    expect(container.firstChild).toMatchSnapshot()
  })
})
