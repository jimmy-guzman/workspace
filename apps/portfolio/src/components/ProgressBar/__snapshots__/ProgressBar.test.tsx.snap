// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBar /> should render default snapshot 1`] = `
<div
  class="progress-bar"
>
  <div
    class="indeterminate"
  />
</div>
`;
